docs/topics/OAuth2.md

```diff
@@ -427,9 +458,10 @@ Returns info about the current authorization. Requires authentication with a bea
     "expires": "2021-01-23T02:33:17.017000+00:00",
     "user": {
         "id": "268473310986240001",
-        "username": "Discord",
+        "username": "discord",
```

Comment:
wouldn't it be better to use a more generic name for the example object here, since "discord" is a banned username

Reply:
I think discord should be kept as example.
It feels fitting for the docs.

But I think that is a discussion worth an issue instead.
There're probably more blocks in the docs with discord as name.

Reply:
eh something you can't encounter because of anti abuse mechanisms shouldn't feel fitting

Reply:
I'd say make an issue out of that comment. I won't be addressing this here.